src/Open3D/Visualization/Visualizer/GuiVisualizer.h

```diff
@@ -62,7 +62,7 @@ class GuiVisualizer : public gui::Window {
 
     bool SetIBL(const char* path);
 
-    bool LoadGeometry(const std::string& path);
+    void LoadGeometry(const std::string& path);
```

Comment:
/// This will load Geometry asynchronously, it will return immediately
(maybe we should call it QueueLoadGeometry?)